lib/puppet/provider/f5_snat/rest.rb

```diff
@@ -0,0 +1,95 @@
+require 'puppet/provider/f5'
+require 'json'
+
+Puppet::Type.type(:f5_snat).provide(:rest, parent: Puppet::Provider::F5) do
+
+  def self.instances
+    instances = []
+    profiles = Puppet::Provider::F5.call_items('/mgmt/tm/ltm/snat')
+    return [] if profiles.nil?
+
+    profiles.each do |profile|
+      full_path_uri = profile['fullPath'].gsub('/','~')
```

Comment:
What's this then used for?

Reply:
@alexjfisher Not sure about the question, can you elaborate it?

Reply:
It's a variable local to instances and is not used.  If you were to run rubocop, it would say...
> Useless assignment to variable - `full_path_uri`.